src/api/app.py

```diff
@@ -95,13 +98,20 @@ async def proxy(request: Request, path: str):
         headers["Authorization"] = f"Bearer {access_token}"
```

Comment:
The body of this `proxy` method is getting a little too long for my liking, let's factor out a new method `handle_proxy_request(request: Request, path: string):`